src/official-widgets/camera-search/camera-search.spec.tsx

```diff
@@ -901,4 +901,139 @@ describe('camera-search', () => {
 
     expect(counter).toEqual(3);
   });
+
+  it('uploading image should return cropped image', async () => {
```

Comment:
Probably no need for separate test case here; just that all the initial searches in this test file should be changed to return the boxed response, just like how they were changed in the ESR test file.